akka-projection-core/src/main/scala/akka/projection/scaladsl/Projection.scala

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 2019-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.scaladsl
+
+import akka.Done
+import akka.stream.{ KillSwitch, KillSwitches, Materializer }
+import akka.stream.scaladsl.{ Keep, Sink, Source }
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+trait Projection {
+  def start()(implicit ex: ExecutionContext, materializer: Materializer): Unit
+  def stop()(implicit ex: ExecutionContext): Future[Done]
+}
```

Comment:
Super reduced Projection API. It's just a wrapper around something that will be managed by the toolkit. 

Implementations will need to define the factories to build projections according to specific needs.

Reply:
There is one important thing missing here that was available on previous examples. There is no `processEnvelope` method. This was added to facilitate instrumentation, but I remove it for now since it won't make sense for a pure Alpakka Kafka projection (see sample).